src/modules/GroupChannel/components/GroupChannelUI/GroupChannelUIView.tsx

```diff
@@ -62,7 +62,9 @@ export const GroupChannelUIView = (props: GroupChannelUIViewProps) => {
   const sdkError = stores?.sdkStore?.error;
   const { logger, isOnline } = config;
 
-  if (loading) {
+  // Note: This is not a loading status of the message list.
+  //  It is just for custom props from the Channel module and is not used in the GroupChannel module. (We should remove this in v4)
```

Comment:
it is included in `ChannelProps` (open public) so we should keep it as is no?

Reply:
Yes, that's correct. It's being kept in ChannelUIProps

https://github.com/sendbird/sendbird-uikit-react/blob/main/src/modules/Channel/components/ChannelUI/index.tsx#L10-L12